content/pages/privacy/policies.json

```diff
@@ -11,6 +11,42 @@
         "updated": "Last updated on",
         "dateFormat": "MM/dd/yyyy"
       }
+    },
+    {
```

Comment:
it's fine if we remove the other languages for now (de, fr, es - as well as the corresponding policies markdown files)